vNext/src/Build.cmd

```diff
@@ -36,25 +43,25 @@ call %_dotnet% restore "%CMDHOME%\.nuget\Tools.csproj"
 @echo Build Debug ==============================
 
 SET CONFIGURATION=Debug
-SET OutDir=%~dp0..\Binaries\%CONFIGURATION%
+SET OutputPath=%~dp0..\Binaries\%CONFIGURATION%
```

Comment:
By setting `OutputPath` instead of `OutDir`, we end up with target-specific output paths. Overriding `OutDir` instead would leave us with a single output directory for both `net462` and `netstandard1.5`, which wont work anymore because we have multiple versions of some DLLs (namely the platform services dll)